Tests/ProfilerTest.php

```diff
@@ -73,9 +73,7 @@ public function testRender()
         // This is only needed for WebProfilerBundle=3.2, remove when support for it is dropped
         $requestCollector = new RequestDataCollector();
```

Comment:
I remember this one, you can remove whole block actually. No need to add $requestCollector to profile

Reply:
Thanks, I removed this 👍